upring.js

```diff
@@ -256,7 +257,7 @@ UpRing.prototype.request = function (obj, callback, _count) {
   }
 
   return this
-}
+})
```

Comment:
Can you call the promisified version requestp, or something different? I do not want request() users to pay the cost of promisified code.